website/docs/r/team_member.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_team_member"
+sidebar_current: "docs-tfe-team-member"
+description: |-
+  Add or remove a user from a team.
+---
+
+# tfe_team_member
+
+Add or remove a user from a team.
+
+~> NOTE on managing team memberships: Terraform currently provides two resources
```

Comment:
_minor_ I'd recommend updating `NOTE` to be `**NOTE:**` to make it stand out

Reply:
Fixed in next commit.